utils/languages.go

```diff
@@ -177,3 +179,49 @@ func DetectLanguage(text string, interfaceLanguage string, acceptLanguage string
 	log.Debug("DetectLanguage: using default language")
 	return consts.SEARCH_LANG_ORDER[interfaceLanguage]
 }
+
+func NumberInHebrew(n int) string {
```

Comment:
Nice. Please add test for this.

Reply:
What is people write:
ט"ז

I mean with quotes... will it work?

Reply:
ט"ז = > works.
I will give Evgeny the task for creating tests. Anyway currently this function works100% ok.